aiida/tools/visualization/graph.py

```diff
@@ -0,0 +1,777 @@
+            "color": "#000000"  # black
+        },
+        LinkType.CALL_CALC: {
+            "style": "dotted",
+            "color": "#000000"  # black
+        },
+        LinkType.CALL_WORK: {
+            "style": "dotted",
+            "color": "#000000"  # black
+        },
+        LinkType.CREATE: {
+            "style": "solid",
+            "color": "#000000"  # black
+        },
+        LinkType.RETURN: {
+            "style": "dashed",
+            "color": "#000000"  # black
+        }
+    }[link_pair.link_type]
+
+    if add_label and not add_type:
+        style['label'] = link_pair.link_label
+    elif add_type and not add_label:
+        style['label'] = link_pair.link_type.name
+    elif add_label and add_type:
+        style['label'] = "{}\n{}".format(link_pair.link_type.name, link_pair.link_label)
+
+    return style
+
+
+def default_node_styles(node):
+    """map a node to a graphviz node style
+
+    :param node: the node to map
+    :type node: aiida.orm.nodes.node.Node
+    :rtype: dict
+
+    """
+    class_node_type = node.class_node_type
+
+    try:
+        default = node.get_style_default()
+    except AttributeError:
+        default = {
+            "shape": "ellipse",
+            "style": "filled",
+            "fillcolor": "#8cd499ff",  # green,
+            "penwidth": 0
+        }
+
+    node_type_map = {
+        "data.code.Code.": {
+            "shape": "ellipse",
+            "style": "filled",
+            "fillcolor": "#4ca4b9aa",  # blue
+            "penwidth": 0
+        },
+        "process.calculation.calcjob.CalcJobNode.": {
+            "shape": "rectangle",
+            "style": "filled",
+            "fillcolor": "#de707fff",  # red
+            "penwidth": 0
+        },
+        "process.calculation.calcfunction.CalcFunctionNode.": {
+            "shape": "rectangle",
+            "style": "filled",
+            "fillcolor": "#de707f77",  # red
+            "penwidth": 0
+        },
+        "process.workflow.workchain.WorkChainNode.": {
+            "shape": "rectangle",
+            "style": "filled",
+            "fillcolor": "#e38851ff",  # orange
+            "penwidth": 0
+        },
+        "process.workflow.workfunction.WorkFunctionNode.": {
+            "shape": "rectangle",
+            "style": "filled",
+            "fillcolor": "#e38851ff",  # orange
+            "penwidth": 0
+        }
+    }
+
+    node_style = node_type_map.get(class_node_type, default)
+
+    return node_style
+
+
+def pstate_node_styles(node):
+    """map a process node to a graphviz node style
+
+    :param node: the node to map
+    :type node: aiida.orm.nodes.node.Node
+    :rtype: dict
+
+    """
+    class_node_type = node.class_node_type
+
+    default = {"shape": "rectangle", "pencolor": "black"}
+
+    process_map = {
+        "process.calculation.calcjob.CalcJobNode.": {
+            "shape": "ellipse",
+            "style": "filled",
+            # "pencolor": "black",
```

Comment:
If we are not using the pencolor, maybe get rid of the commented out code

Reply:
fixed in 6e599e9